pcsx2/CMakeLists.txt

```
@@ -85,9 +85,12 @@ if(WIN32)
 	)
 	target_compile_definitions(PCSX2 PRIVATE
 		_M_SSE=0x401 # TODO: Multiple ISA
+		WINVER=0x0602
+		_WIN32_WINNT=0x0602
```

```suggestion
		WINVER=0x0603
		_WIN32_WINNT=0x0603
```

https://docs.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=msvc-160
> #define _WIN32_WINNT_WIN8                   0x0602 // Windows 8
> #define _WIN32_WINNT_WINBLUE                0x0603 // Windows 8.1